### PR TITLE
PR for old Attribute Stuff

### PR DESCRIPTION
**Pull request by StrangeOne101**
~~I'm aware it can't be merged but I also cant see the issue so I'm PRing it in hope I can see the problem after~~ Fixed

PR'd the stuff you asked for months ago now lol

### CODE REVIEW

**Comment by StrangeOne101** [comment]
I love GitHub. I can fix the issues online

File: src/com/jedk1/jedcore/ability/waterbending/FrostBreath.java [hunk 1]
```diff
@@ -47,6 +47,7 @@
 			Biome.WARPED_FOREST,
 			Biome.BADLANDS,
 			Biome.BADLANDS_PLATEAU,
+			Biome.WOODED_BADLANDS_PLATEAU,
 			Biome.ERODED_BADLANDS,
 			Biome.WOODED_BADLANDS_PLATEAU,
```
Comment (Aztlon) [inline src/com/jedk1/jedcore/ability/waterbending/FrostBreath.java]: Could you delete this so there aren't two of the same biome in this list? (Wooded Badlands Plateau)

**Review by Aztlon — CHANGES_REQUESTED** [review]
Slight change requested in FrostBreath, looks good otherwise

**Comment by StrangeOne101** [comment]
No idea how that got in there. Either way, removed

**Review by Aztlon — APPROVED** [review]

**Pull request merged**